stl/inc/memory

```diff
@@ -3184,7 +3186,7 @@ public:
         _Check_load_memory_order(_Order);
         shared_ptr<_Ty> _Result;
         const auto _Rep = this->_Lock_and_load();
-        _Result._Ptr    = this->_Ptr;
+        _Result._Ptr    = this->_Ptr.load(memory_order::relaxed);

```

Comment:
I observe (again?) that we're accumulating inconsistency about `memory_order::MEOW` versus `memory_order_MEOW`. No change requested as we're already inconsistent.

Reply:
Changed to use old names